testsuite/features/core/container_create_repository.feature

```diff
@@ -0,0 +1,121 @@
+    When I follow the left menu "Software > Manage > Channels"
+    And I follow "Fake-i586-Channel"
+    And I enter "file:///etc/pki/rpm-gpg/uyuni-tools-gpg-pubkey-0d20833e.key" as "GPG key URL"
+    And I click on "Update Channel"
+    Then I should see a "Channel Fake-i586-Channel updated" text
+    When I follow "Repositories" in the content area
+    And I select the "fake-i586-repo" repo
+    And I click on "Save Repositories"
+    Then I should see a "Fake-i586-Channel repository information was successfully updated" text
+
+  Scenario: Synchronize the repository in the i586 channel
+    When I follow the left menu "Software > Manage > Channels"
+    And I follow "Fake-i586-Channel"
+    And I follow "Repositories" in the content area
+    And I follow "Sync"
+    And I wait at most 60 seconds until I do not see "Repository sync is running." text, refreshing the page
+    And I click on "Sync Now"
+    Then I should see a "Repository sync scheduled for Fake-i586-Channel." text
+
+@deblike_minion
+  Scenario: Add a test repository for Debian-like
+    When I follow the left menu "Software > Manage > Repositories"
+    And I follow "Create Repository"
+    And I enter "fake-debian-repo" as "label"
+    And I select "deb" from "contenttype"
+    And I enter "http://localhost/pub/TestRepoDebUpdates/" as "url"
+    And I click on "Create Repository"
+    Then I should see a "Repository created successfully" text
+
+@deblike_minion
+  Scenario: Add the Debian-like repository to the AMD64 channel
+    When I follow the left menu "Software > Manage > Channels"
+    And I follow "Fake-Deb-AMD64-Channel"
+    And I follow "Repositories" in the content area
+    And I select the "fake-debian-repo" repo
+    And I click on "Save Repositories"
+    Then I should see a "Fake-Deb-AMD64-Channel repository information was successfully updated" text
+
+@deblike_minion
+  Scenario: Synchronize the Debian-like repository in the AMD64 channel
+    When I follow the left menu "Software > Manage > Channels"
+    And I follow "Fake-Deb-AMD64-Channel"
+    And I follow "Repositories" in the content area
+    And I follow "Sync"
+    And I wait at most 60 seconds until I do not see "Repository sync is running." text, refreshing the page
+    And I click on "Sync Now"
+    Then I should see a "Repository sync scheduled for Fake-Deb-AMD64-Channel." text
+
+@rhlike_minion
+  Scenario: Add the repository to the RedHat-like channel
+    When I follow the left menu "Software > Manage > Channels"
+    And I follow "Fake-RH-Like-Channel"
+    And I enter "file:///etc/pki/rpm-gpg/uyuni-tools-gpg-pubkey-0d20833e.key" as "GPG key URL"
+    And I click on "Update Channel"
+    Then I should see a "Channel Fake-RH-Like-Channel updated" text
+    When I follow "Repositories" in the content area
+    And I select the "fake-rpm-repo" repo
+    And I click on "Save Repositories"
+    Then I should see a "Fake-RH-Like-Channel repository information was successfully updated" text
+
```

Comment:
The cleanup to disable source package syncing should probably be added, it might cause some issues down the line: https://github.com/uyuni-project/uyuni/blob/master/testsuite/features/core/srv_create_repository.feature#L162-L163